application/lib/view/widgets/offers_horizontal_list.dart

```diff
@@ -0,0 +1,52 @@
+import 'package:application/model/offer_model.dart';
+import 'package:application/utils/translation_en.dart';
+import 'package:application/view/screens/offer_screen.dart';
+import 'package:application/view/widgets/offer_item.dart';
+import 'package:flutter/material.dart';
+
+class OffersHorizontalList extends StatelessWidget {
+  final List<OfferModel> offers;
```

Comment:
Legyen egy bemenő bool paraméter hogy isSmallSizeRequired. 
Ez alapján állítsuk be az offerItem-hez a width és a font size paramétereket. Az inline if operátorral: isSmallSizeRequired ? 300 : 150